From ef30b5f7bf29c58440fc10cc75119501d64b1882 Mon Sep 17 00:00:00 2001
From: Bai Ping <ping.bai@nxp.com>
Date: Mon, 11 Jan 2016 14:16:00 +0800
Subject: [PATCH 1402/1691] MLK-11987 cpufreq: imx: remove unused value
 assignment

commit ef30b5f7bf29c58440fc10cc75119501d64b1882 from
git://git.freescale.com/imx/linux-2.6-imx.git imx_4.1.15_1.0.0_ga

Remove the unused value assignment reported by
coverity check(CID 17279).

Signed-off-by: Bai Ping <ping.bai@nxp.com>
---
 drivers/cpufreq/imx6q-cpufreq.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/cpufreq/imx6q-cpufreq.c b/drivers/cpufreq/imx6q-cpufreq.c
index e03d016..14b3040 100644
--- a/drivers/cpufreq/imx6q-cpufreq.c
+++ b/drivers/cpufreq/imx6q-cpufreq.c
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2013-2015 Freescale Semiconductor, Inc.
+ * Copyright (C) 2013-2016 Freescale Semiconductor, Inc.
  *
  * This program is free software; you can redistribute it and/or modify
  * it under the terms of the GNU General Public License version 2 as
@@ -137,7 +137,7 @@ static int imx6q_set_target(struct cpufreq_policy *policy, unsigned int index)
 		if (freq_hz > clk_get_rate(pll2_pfd2_396m_clk))
 			clk_set_parent(secondary_sel, pll2_bus);
 		else
-			ret = clk_set_parent(secondary_sel, pll2_pfd2_396m_clk);
+			clk_set_parent(secondary_sel, pll2_pfd2_396m_clk);
 		clk_set_parent(step_clk, secondary_sel);
 		clk_set_parent(pll1_sw_clk, step_clk);
 	} else {
-- 
1.9.1

